understanding/keyboard.html

```diff
@@ -55,248 +53,177 @@
                   </dd>
                   
-                  <dt>Example 3: Moving between and connecting discrete points</dt>
+                  <dt>事例 3: 離れている点の間の移動及び接続</dt>
                   
-                  <dd>A connect-the-dots program allows the user to move between dots on a screen and use
-                     the spacebar to connect the current dot to the previous one.
+                  <dd>点を結ぶプログラムは、利用者が画面上の点間を移動し、スペースキーで現在の点と直前の点を結ぶことを可能にする。
                   </dd>
                   
-                  <dt>Example 4: Exception - Painting Program</dt>
+                  <dt>事例 4: 例外 － お絵描きプログラム</dt>
                   
-                  <dd>A watercolor painting program passes as an exception because the brush strokes vary
-                     depending on the speed and duration of the movements.
+                  <dd>水彩画を描くプログラムは、筆の動きが速度及び持続時間によって変化するため、例外の一つとして認められる。
                   </dd>
                   
-                  <dt>Example 5: Exception - Model helicopter flight training simulator</dt>
+                  <dt>事例 5: 例外 － 模型ヘリコプター飛行訓練シミュレータ</dt>
                   
-                  <dd>A model helicopter flight training simulator passes as an exception because the nature
-                     of the simulator is to teach real-time behavior of a model helicopter.
+                  <dd>ヘリコプター飛行訓練シミュレータは、シミュレータの性質上、模型ヘリコプターの動作をリアルタイムで教えるものであるため、例外の一つとして認められる。
                   </dd>
                   
-                  <dt>Example 6: A PDA with an optional keyboard</dt>
+                  <dt>事例 6: オプションのキーボード付き PDA</dt>
                   
-                  <dd>A PDA device that is usually operated via a stylus has an optional keyboard that can
-                     be attached.  The keyboard allows full Web browsing in standard fashion.  The Web
-                     content is operable because it was designed to work with keyboard-only access.
+                  <dd>通常スタイラスペンで操作する PDA 機器は、オプションで取り付け可能なキーボードがある。そのキーボードは、標準的な方法で完全なウェブ閲覧を可能にする。キーボードのみのアクセスで動作するように設計されているので、ウェブコンテンツは操作可能である。
                   </dd>
                   
-                  <dt>Example 7: Simple search form with pointer-operable submit button</dt>
+                  <dt>事例 7: ポインタ操作可能な送信ボタンを備えたシンプルな検索フォーム</dt>
                   
-                  <dd>A search form includes a text input field followed by a submit button. The submit
-                     button itself
-                     has been coded so that it does <em>not</em> receive focus, and can only be activated using a pointer input.
-                     However, since keyboard users can submit the search by pressing <kbd>Enter</kbd> in the text input
-                     after typing their search terms, the form passes this Success Criterion.
+                  <dd>検索フォームには、テキスト入力フィールドとそれに続く送信ボタンがある。送信ボタン自体はフォーカスを受け取ら<em>ない</em>ようにコーディングされており、ポインタ入力によってのみ実行できるようになっている。ただしキーボード利用者は、テキスト入力で検索語を入力した後に <kbd>Enter</kbd> キーを押すことで検索を送信できるので、このフォームは達成基準に合格となる。
```

Comment:
このhoweverは「しかし」とするのがしっくりくると思います。

```suggestion
                  <dd>検索フォームには、テキスト入力フィールドとそれに続く送信ボタンがある。送信ボタン自体はフォーカスを受け取ら<em>ない</em>ようにコーディングされており、ポインタ入力によってのみ実行できるようになっている。しかし、キーボード利用者は、テキスト入力で検索語を入力した後に <kbd>Enter</kbd> キーを押すことで検索を送信できるので、このフォームは達成基準に合格となる。
```